ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/CCDCase.java

```diff
@@ -26,4 +27,6 @@
     private String defendantEmail;
     private CCDResponse response;
     private String respondedAt;
```

Comment:
Can we change the type to `LocalDateTime` here?

Reply:
LocalDateTime is not supported at ccd at the moment, https://tools.hmcts.net/confluence/display/RCCD/CCD+Supported+Field+Types

Reply:
Yeah, but this code is on our side. Jackson will serialize field value to a string which gets sent over to CCD. I think it should be possible to have a more precise type here.

Reply:
done